toonz/sources/toonz/viewerpane.cpp

```diff
@@ -588,6 +600,12 @@ void SceneViewerPanel::onSceneChanged() {
   TApp *app         = TApp::instance();
   ToonzScene *scene = app->getCurrentScene()->getScene();
   assert(scene);
+  m_flipConsole->setFrameRate(TApp::instance()
+                                  ->getCurrentScene()
+                                  ->getScene()
+                                  ->getProperties()
+                                  ->getOutputProperties()
+                                  ->getFrameRate());
```

Comment:
I'm not looking at the code, so I may be mistaken, but I believe the combo viewer does it- if I change the scene fps, the combo viewer fps updates (but not the viewer).  This is handy and less confusing if you change the scene fps and wonder why the viewer is still playing at the old fps.